app/src/main/java/com/tr4n/moviedb/ui/detail/MovieDetailFragment.kt

```diff
@@ -0,0 +1,41 @@
+package com.tr4n.moviedb.ui.detail
+
+import android.view.View
+import android.widget.Toast
+import com.tr4n.moviedb.R
+import com.tr4n.moviedb.base.BaseFragment
+import kotlinx.android.synthetic.main.activity_main.*
+import kotlinx.android.synthetic.main.fragment_movie_detail.*
+
+
+/**
+ * An example full-screen fragment that shows and hides the system UI (i.e.
+ * status bar and navigation/system bar) with user interaction.
+ */
+class MovieDetailFragment(private val movieId: Long) : BaseFragment(R.layout.fragment_movie_detail) {
```

Comment:
Không truyền qua constructor 
truyền qua hàm newInstance rồi em tạo bundle chứ 
```suggestion
class MovieDetailFragment(private val movieId: Long) : BaseFragment(R.layout.fragment_movie_detail) {
```